applications/LS-Dyna-stampede3/app.json

```diff
@@ -0,0 +1,124 @@
+{
+    "id": "LS-Dyna-stampede3",
+    "version": "2024R1",
```

Comment:
Just curious, where is the version referenced from?

Reply:
It uses the ANSYS 2024R1 build